deployment/playground-app.yaml

```diff
@@ -129,6 +129,19 @@ spec:
       targetPort: 80
 ---
 apiVersion: v1
+kind: Service
+metadata:
+  name: flagd-service
+  namespace: open-feature-demo
+spec:
+  type: ClusterIP
+  selector:
+    app: open-feature-demo
+  ports:
+    - port: 80
+      targetPort: 80
+---
+apiVersion: v1
```

Comment:
This service is references by the port forward, but didn't seem to be defined anywhere.